hail_search/web_app.py

```diff
@@ -97,6 +98,17 @@ async def multi_lookup(request: web.Request) -> web.Response:
     return web.json_response({'results': result}, dumps=hl_json_dumps)
 
 
+async def reload_globals(request: web.Request) -> web.Response:
+    await sync_to_async_hail_query(request, lambda _: load_globals())
+    return web.json_response(
```

Comment:
I feel like it would be neater to have `load_globals` return this, rather than need to separately compute it